Sources/Datadog/DDRUMMonitor.swift

```diff
@@ -79,13 +79,15 @@ public class DDRUMMonitor {
     /// Notifies that an Error occurred in currently presented View.
     /// - Parameters:
     ///   - message: a message explaining the Error.
+    ///   - type: the type of Error.
```

Comment:
It's not super clear to me what a "type of Error" is, or how one would be expected to use it. I think it'd help to add least add an example for context here. :)

Reply:
I kept the documentation in line with other parts of the SDK for now!